plugins/repository-azure/build.gradle

```diff
@@ -369,17 +370,6 @@ tasks.register("azureThirdPartyTest", Test) {
 }
 tasks.named("check").configure { dependsOn("azureThirdPartyTest") }
 
-// test jar is exported by the integTestArtifacts configuration to be used in the encrypted Azure repository test
-configurations {
-  internalClusterTestArtifacts.extendsFrom internalClusterTestImplementation
-  internalClusterTestArtifacts.extendsFrom internalClusterTestRuntime
-}
-
-def internalClusterTestJar = tasks.register("internalClusterTestJar", Jar) {
-  appendix 'internalClusterTest'
-  from sourceSets.internalClusterTest.output
-}
-
-artifacts {
-  internalClusterTestArtifacts internalClusterTestJar
+testArtifacts {
```

Comment:
Can we move this config higher up in the build file? Seems better to not have this oddly buried down here.

Reply:
Moved those just below the dependency declarations